packages/cardano-services/test/ChainHistory/DbSyncChainHistoryProvider/__snapshots__/ChainHistoryBuilder.test.ts.snap

```diff
@@ -99,7 +99,7 @@ Map {
 
 exports[`ChainHistoryBuilder queryProtocolParams query protocol parameters from last epoch 1`] = `
 Object {
-  "coinsPerUtxoWord": NaN,
+  "coinsPerUtxoByte": NaN,
```

Comment:
The value isn't changing, so it's not part of this PR's scope, but let's make sure to log this for follow-up work